src/lighteval/tasks/prompt_manager.py

```diff
@@ -234,7 +235,7 @@ def _single_turn_context(
                     toks = [self.model.tok_encode(msg["content"]) for msg in output]
                     toks = [t for ts in toks for t in ts]
 
-        if isinstance(self.model, LiteLLMClient):
+        if type(self.model) in [LiteLLMClient, InferenceProvidersClient]:
```

Comment:
```suggestion
        if isinstance(self.model, (LiteLLMClient, InferenceProvidersClient)):
```

(a bit late to the party but this is more robust in Python to check if object is instance of one from a list)

Reply:
Oh nice, didn't know you could do isinstance from a list :fire: